bundles/org.openhab.binding.shelly/src/main/java/org/openhab/binding/shelly/internal/discovery/ShellyBasicDiscoveryService.java

```diff
@@ -131,14 +132,12 @@ public void deactivate() {
             api.initialize();
             devInfo = api.getDeviceInfo();
             mac = getString(devInfo.mac);
-            model = devInfo.type;
+            model = getString(devInfo.type);
             auth = getBool(devInfo.auth);
-            if (name.isEmpty() || name.startsWith("shellyplusrange")) {
+            if (name.isEmpty() || name.startsWith(SERVICE_NAME_SHELLYPLUSRANGE_PREFIX)) {
```

Comment:
```suggestion
            if (name.isBlank() || name.startsWith(SERVICE_NAME_SHELLYPLUSRANGE_PREFIX)) {
```

Reply:
Not related, but a whitespace would not go through the code block, while i think it should. Nevertheless i can understand you don;t want this before code freeze.